src/output.cpp

```diff
@@ -1194,15 +1194,19 @@ std::string format_item_info( const std::vector<iteminfo> &vItemDisplay,
     return buffer.str();
 }
 
-int draw_item_info(WINDOW *win, const std::string sItemName,
+int draw_item_info(WINDOW *win, const std::string sItemName, const std::string sTypeName,
                    std::vector<iteminfo> &vItemDisplay, std::vector<iteminfo> &vItemCompare,
                    int &selected, const bool without_getch, const bool without_border, const bool handle_scrolling)
 {
     std::ostringstream buffer;
     int line_num = 1;
-    if (sItemName != "") {
-        buffer << sItemName << "\n \n"; //This space is required, otherwise it won't make an empty line.
+	if (sItemName != "") {
+		buffer << sItemName << "\n";
+	}
```

Comment:
There are tabs here, and in iuse_actor.cpp. Those are not allowed, please replace them with 4 spaces per tab.